integration/stability/soak_parallel_rm.sh

```diff
@@ -33,7 +33,7 @@ RUNTIME_PATH=$(command -v $RUNTIME)
 # The place where virtcontainers keeps its active pod info
 # This is ultimately what 'kata-runtime list' uses to get its info, but
 # we can also check it for sanity directly
-VC_POD_DIR="${VC_POD_DIR:-/var/lib/vc/sbs}"
+VC_POD_DIR="${VC_POD_DIR:-/run/vc/sbs}"
```

Comment:
I was going to suggest we could change the code to get the directory from `kata-runtime kata-env` - but, I don't think we print that information (our workdir) in kata-env.... maybe we should? :-)